### DISCUSSION TITLE
Is it possible to do publicKeyTweakMul with noble-curves library?

### DISCUSSION THREAD

**Discussion by charlie632** [paulmillr/noble-curves#36] [Q&A] [answered]
Hey there, 

I'm porting a library that uses the [secp256k1-node](https://github.com/cryptocoinjs/secp256k1-node) Library, which only works in NodeJS, to something that works on the browser. I found this amazing library that has been working pretty well. However, the original implementation uses the [secp256k1.publicKeyTweakMul](https://github.com/cryptocoinjs/secp256k1-node/blob/6d3474b81d073cc9c8cc8cfadb580c84f8df5248/lib/index.js#L196) function for something, and I didn't find a method for doing this. Is it possible to do this with this library?

If it is impossible, can anyone help me or guide me on how to implement this method?

Thank you!

**Comment by paulmillr** [comment, marked as answer]
There is a secp256k1-node compatibility layer available in a separate repository. Feel free to use its code:

https://github.com/ethereum/js-ethereum-cryptography/blob/d316d2b22b5cd4746cc9324e0c5a94c7d033c61c/src/secp256k1-compat.ts#L245-L263

**Reply by charlie632** [reply]
Amazing! Thank you for your time and the work you put into this project.
